src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -40,12 +40,19 @@ public void verifyNoSlotPresent(String slotName) {
   }
 
   public void verifySlotExpanded(String slotName) {
+    By slotById = By.id(slotName);
+    WebElement slot = driver.findElement(slotById);
+
+    wait.forElementVisible(slotById);
```

Comment:
Shouldn't we do this check after scrolling on mobile? (one line below)

Reply:
Yea, it makes sense. However it works right now and that's because in the test we're scrolling to recirculation pre-footer: https://github.com/Wikia/selenium-tests/blob/8ba8965b86e62520215a0640b8bb99bdf071da8c/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java#L88 I'll change it anyway. Thanks for catching 🙇